src/main.js

```diff
@@ -31,9 +31,9 @@ ipfsInstance.setBinPath(ipfsPath)
 const ipfsInitPath = path.join(app.getPath('userData'), 'ipfs-init')
 ipfsInstance.setIpfsFolder(ipfsInitPath)
 
-async function loadAragonClient (network = 'main') {
-  const latestHashForNetwork = await getLatestFromRepo('aragon.aragonpm.eth', network)
-  await pinAragonClientForNetwork(latestHashForNetwork, network)
+async function loadAragonClient () {
```

Comment:
We should also change [this usage](https://github.com/aragon/aragon-desktop/blob/658c7e5aab0e23aba09c42aea7ea9cba4095f82b/src/main.js#L150) of this function

Reply:
Done!